### ISSUE TITLE
Calculate GXG from MF6 HDS file not working

### ISSUE THREAD

**Issue by WouterSwierstra** [Deltares/imod-python#1314]
**Bug description**
When using iMOD python to calculate GXG from a Modflow6 HDS file, I get an error message. See example.
Plus I noticed that if the xarray datamodel  for the heads contains whole years, the GHG and GLG are calculated over the hydrological year (number of years in datamodel minus 1) and the GVG over all the years in the xarray datamodel. See example.

**Steps to reproduce**
1. What error message do you get?
     ValueError: Object has inconsistent chunks along dimension bimonth. This can be fixed by calling unify_chunks().
2. Can you create a minimum reproducible example?    - Can you create a small script that reproduces the error?
[link (experis 20250101)](https://corporateroot-my.sharepoint.com/:u:/g/personal/wouter_swierstra_rhdhv_com/EX39FXPDcBVJjC28FVIXdFEB8U8X8rDpQivn60kxlST2FA?e=g2DlKR)
3. Code used:
     head = imod.mf6.open_hds(path_hds, path_grb)
    if start_date is not None:
        timedelta = pd.to_timedelta(head["time"].values, unit="D")
        head["time"] = parse_time(start_date) + timedelta
    else:
        print('startdatum moet opgegeven worden')
        os.sys.exit(1)

    heads = head.sel(time=(head.time.dt.year >= 2005) & (head.time.dt.year <= 2006), layer=1)
    gxg = imod.evaluate.calculate_gxg(head.sel(layer=1))

**Desktop (please complete the following information):**
 - OS:  Windows 11
 - iMOD Python Version [e.g. 0.18.1]

**Comment by JoerivanEngelen** [comment]
Hi @WouterSwierstra,

It appears that there are have been some changes in xarray/dask behavior, which we didn't catch in our tests.
I've got a fix that seems to fix this in PR: . I'm not entirely sure if it causes performance problems. Inferring from the comment left by @Huite, it appears it should do the right thing. I tested it on your dataset with your script and it makes things work. 

Regarding the years used for the GVG / and GXGs calculations, you are right that this can differ. From the docstrings:

```
    Hydrological years without all 24 14/28 dates present are discarded for glg and ghg.
    Years without the 3 dates for gvg are discarded.
```

It can therefore be that the start and/or final year of your simulation does not contain all 24 dates and is therefore discarded, whereas for the gvg these are preserved. This is intentionally programmed that way. Does this fully answer your question? Let me know if you disagree, or have an edge case where ``calculate_gxg`` is showing inappropriate behavior.
